contrib/cmake/src/api/python/CMakeLists.txt

```diff
@@ -60,13 +62,23 @@ add_custom_command(OUTPUT "${z3py_bindings_build_dest}/z3consts.py"
   COMMENT "Generating z3consts.py"
   ${ADD_CUSTOM_COMMAND_USES_TERMINAL_ARG}
 )
-list(APPEND build_z3_python_bindings_target_depends "${z3py_bindings_build_dest}/z3consts.py")
+list(APPEND build_z3_python_bindings_target_depends "${z3py_bindings_build_dest}/z3/z3consts.py")
+
+# Link libz3 into the python directory so bindings work out of the box
+add_custom_command(OUTPUT "${z3py_bindings_build_dest}/libz3${CMAKE_SHARED_MODULE_SUFFIX}"
+  COMMAND "${CMAKE_COMMAND}" "-E" "create_symlink"
```

Comment:
Seems not working on windows:

C:\py\z3\build>ninja
[1/1] Linking libz3 into python directory
FAILED: cmd.exe /C "cd /D C:\py\z3\build\src\api\python && "C:\Program Files (x86)\CMake\bin\cmake.exe" -E create_symlink C:/py/z3/build/libz3.dll C:/py/z3/build/python/libz3.dll"
failed to create symbolic link 'C:/py/z3/build/python/libz3.dll': No error
ninja: build stopped: subcommand failed.